### DISCUSSION TITLE
Followed instructions however certification not showing for localhost

### DISCUSSION THREAD

**Discussion by bastianhilton** [FiloSottile/mkcert#443] [Q&A] [answered]
Hello i followed the instructions on the github page for this project using Windows installation with choco. It installed, i successfully installed mkcert to my store however when i go to localhost, it doesn't show https and even when i type in https it doesn't recognize a cert.

I'm using:

Windows 10 Pro (ver 21H1)
Installed mkcert with (Microsoft Terminal in administrator) through chocolatey. 
Using Windows Edge (Chromium version)

I restarted my browser and still nothing, could someone point me in the right direction of getting this working.

**Comment by minecraftchest1** [comment, marked as answer]
Did you configure the server to use the cert? mkcert won't do that automatically. Also, please send a screenshot of when it doesn't recognize the cert.
